v8go.cc

```diff
@@ -1207,6 +1207,19 @@ RtnValue FunctionNewInstance(ValuePtr ptr, int argc, ValuePtr args[]) {
   return rtn;
 }
 
+RtnValue FunctionSourceMapUrl(ValuePtr ptr) {
+  LOCAL_VALUE(ptr)
+  RtnValue rtn = {nullptr, nullptr};
+  Local<Function> fn = Local<Function>::Cast(value);
+  MaybeLocal<Value> result = fn->GetScriptOrigin().SourceMapUrl();
```

Comment:
It looks like [ScriptOrigin::SourceMapUrl](https://v8docs.nodesource.com/node-16.0/db/d84/classv8_1_1_script_origin.html#a1d2ec57af7279bd782132fcdff522bc6) returns a `Local<Value>`.  Is that just getting implicitly cast to `MaybeLocal<Value>`?  If so, then we should get rid of that implicit cast to make the code clearer and avoid unnecessary error checking.

Reply:
I think that's the pattern of these functions for anything that can return Local be cast to MaybeLocal and be checked for emptiness so the `RtnValue` can get an error instead of the other fields set.

Reply:
It wouldn't ever be empty if v8 returns a Local instead of a MaybeLocal, so the Function.SourceMapUrl should reflect this too.  Any error checking introduced to handle this cast is just dead code that adds confusion all the way up to the code using the library.